src/test/java/posters/pageobjects/pages/checkout/PlaceOrderPage.java

```diff
@@ -107,15 +106,18 @@ public void validateProduct(int position, String productName, int productCount,
     public void validateAddressesAndPayment(Address shippingAddress, Address billingAddress, CreditCard creditcard)
     {
         // Shipping address
-        // Name
-        // Makes sure the shipping address name matches the parameter
-        shippingAddressForm.find(".name").shouldHave(exactText(shippingAddress.getFullName()));
+        // fullName
+        // Makes sure the shipping address fullName matches the parameter
+        String firstName = shippingAddress.getFirstName();
+        String lastName = shippingAddress.getLastName();
+        String fullName = firstName + " " + lastName;
+        shippingAddressForm.find(".name").shouldHave(exactText(fullName));
```

Comment:
@oomelianchuk It would be nice to rename the  `shippingAddressForm` to `shippingAddressContainer` or `shippingAddressTile` to avoid misunderstandings.

Reply:
Done